hw/ip/otbn/util/otbn-objdump

```diff
@@ -137,11 +137,12 @@ def main() -> int:
     has_disasm = snoop_disasm_flags(args)
 
     objdump = find_tool('objdump')
-    cmd = [objdump] + args
     try:
         if not has_disasm:
+            cmd = [objdump] + args
             return subprocess.run(cmd).returncode
         else:
+            cmd = [objdump, '-M', 'numeric'] + args
```

Comment:
While you're at it, you also might want to add `-Mno-aliases`, since we don't define most aliases in our specification.

Reply:
I'm not sure: we have an `li` pseudo-instruction, which was dissassembling nicely :-) Are we happy to lose that?

Reply:
No strong requirement either way, but there are a number of other pseudo-instructions in RV32 which we don't specify in OTBN (like nop, ret, etc.); we could either specify those as well, or risk people getting a disassembler output which they cannot map to the OTBN spec.

Reply:
Well, actually we do specify `nop` and `ret` :-) (`li`, `nop` and `ret` are the only three pseudo-instructions we currently support).

Looking at table 25.3 of the ISA spec, it looks like there are quite a few we don't. I'll add `no-aliases` for now; we can always come back to it later.